### DISCUSSION TITLE
Where to find rule ids?

### DISCUSSION THREAD

**Discussion by dot-mike** [ansible/ansible-lint#2276] [Q&A] [answered]
The ansible lint documentation says I can skip rules for specific lines by using `#noqa [rule_id]`, but I'm having a hard time finding rule_id documented in the documentation. Example from the documentation: [False Positives: Skipping Rules](https://ansible-lint.readthedocs.io/en/latest/rules/#false-positives-skipping-rules).

If I get the following warning: `ansible/playbook.yml:83: command-instead-of-module git used in place of git module`, and I go to the documentation for that rule it does not say what the rule_id is: [command-instead-of-module](https://ansible-lint.readthedocs.io/en/latest/default_rules/#command-instead-of-module).

**Comment by ssbarnea** [comment, marked as answer]
Yes, that is the rule id (identifier): `command-instead-of-module`.

**Reply by dot-mike** [reply]
Got it, thanks
